src/utils/gh-auth.js

```diff
@@ -1,92 +1,92 @@
 // A simple ghauth inspired library for getting a personal access token
 const http = require('http')
+const os = require('os')
 const process = require('process')
 const querystring = require('querystring')
 
 const { Octokit } = require('@octokit/rest')
+const dotProp = require('dot-prop')
 const getPort = require('get-port')
 const inquirer = require('inquirer')
-const get = require('lodash/get')
+
+const { version } = require('../../package.json')
 
 const { createDeferred } = require('./deferred')
 const openBrowser = require('./open-browser')
 
 const SERVER_PORT = 3000
+const USER_AGENT = `Netlify CLI ${version}`
 
-module.exports = async function getGitHubToken({ opts, log }) {
-  log('')
-
-  opts = {
```

Comment:
`gh-auth` is only used in `config-github.js` so no point of passing `opts` (the same options are used all the time)